src/main/java/cn/nukkit/block/BlockIcePacked.java

```diff
@@ -2,6 +2,7 @@
 
 import cn.nukkit.item.Item;
 import cn.nukkit.item.ItemTool;
+import cn.nukkit.utils.BlockColor;
```

Comment:
Why?

Reply:
Eu implementei a cor e depois vi a extensão e não retirei a importação. kkkkkkkk

Reply:
I implemented the color and then saw the extension and didn't remove the import. kkkkkk